source/_components/media_player.vizio.markdown

```diff
@@ -70,34 +86,40 @@ media_player:
   - platform: vizio
     host: IP_ADDRESS
     access_token: AUTH_TOKEN
+    device_type: tv
```

Comment:
Please remove line 89

Reply:
Remove it or change it to device_class?

Reply:
Remove because it is not a required config variable.